tiltak/src/main/kotlin/no/nav/amt/tiltak/tiltak/repositories/GjennomforingRepository.kt

```diff
@@ -60,12 +58,9 @@ open class GjennomforingRepository(private val template: NamedParameterJdbcTempl
 				:fremmoteDato)
 	""".trimIndent()
 
-		val id = UUID.randomUUID()
```

Comment:
Hadde i utgangspunktet sett for meg at man beholder den interne id'en som før og legger til den andre som en id til eksternt system, tilsvarende det som var hvor arenaId er id til det eksterne systemet

Reply:
Tror vi har gått litt vekk fra og ha både intern og ekstern id. Tanken bak å sette IDen i ACLen er at når vi får ting fra andre kilder som f.eks valp så vil de også sende med ID. Det gjør det også mye enklere å inserte ting i amt-tiltak siden man trenger ikke å lage ekstra søk for å finne IDen til f.eks gjennomføring når vi skal inserte en ny deltaker.